fuzzers/074-dump_all/ignored_wires.txt

```diff
@@ -0,0 +1,110 @@
+LIOI3_X0Y141/LIOI_I2GCLK_TOP1
```

Comment:
Is there a writeup somewhere on how you came up with this list?

Reply:
It's literally just the wires that currently aren't defined via the tileconn.json.  I'm going to file an issue to figure out how to extract tileconn.json entries to describe them, but my goal right now is to start to integrate the updated database with VPR, rather than fix these 110 wires.

Reply:
Filed #129